common/data.go

```diff
@@ -23,10 +23,11 @@ import (
 )
 
 type DbManager struct {
-	Data      apid.DataService
-	Db        apid.DB
-	DbMux     sync.RWMutex
-	dbVersion string
+	Data          apid.DataService
+	Db            apid.DB
+	DbMux         sync.RWMutex
+	CipherManager CipherManagerInterface
```

Comment:
are all these public instance variables? shouldn't we use opaque structs?

Reply:
Yes public, because it's embedded into `verifyApiKey` and `accessEntity`'s dbManagers.
In short, Its fields are initialized in other packages.